lib/elixir_sense/core/source.ex

```diff
@@ -170,7 +170,7 @@ defmodule ElixirSense.Core.Source do
       |> split_lines
       |> Enum.map_join("\n", fn line ->
         # this is a naive comment strip - it will not honour # in strings, chars etc
-        Regex.replace(~r/[^<]\#.*$/, line, "")
+        Regex.replace(~r/[^<]\#[^{].*$/, line, "")
```

Comment:
```suggestion
        Regex.replace(~r/[^<]\#(?!\{).*$/, line, "")
```
@oo6 your solution will fail to strip `#` when it is the last character on the line as negative character class still requires something. See https://www.regular-expressions.info/lookaround.html for details